src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```

Comment:
This line:
https://github.com/mboudreau/angular.js/blob/fc0ddbf75611dd79da2a19a400dd4e747c290ea6/src/Angular.js#L902
from the top of your `master` branch is still what was here before.

To make sure we're on the same page. It is:
```
 * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```
it should be:
```
 * @param {string=} the name of the library available under `window` to be used for angular.element
```